x-pack/metricbeat/module/ibmmq/status/_meta/data.json

```diff
@@ -0,0 +1,31 @@
+{
+    "@timestamp": "2020-01-07T10:05:34.853Z",
+    "event": {
+        "dataset": "ibmmq.status",
+        "duration": 115000,
+        "module": "ibmmq"
+    },
+    "metricset": {
+        "name": "status",
+        "period": 10000
+    },
+    "prometheus": {
+        "labels": {
+            "handler": "prometheus",
+            "instance": "127.0.0.1:60576",
+            "job": "ibmmq"
+        },
+        "metrics": {
+            "http_request_duration_microseconds_count": 20,
```

Comment:
I'm afraid these are exporter's specific metrics? Could we have a sample with MQ metrics in for better quality in our docs? We can just reuse https://github.com/elastic/beats/pull/15301/files#diff-feefbf30e578ae478841482cad05f326R126 ;)

Reply:
Fixed.